controllers/storagecluster/onboarding_validation_keys_generator_job.go

```diff
@@ -0,0 +1,109 @@
+package storagecluster
+
+import (
+	"fmt"
+	"os"
+
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
```

Comment:
```suggestion
	kerrors "k8s.io/apimachinery/pkg/api/errors"
```
just not to collide w/ stdlib pkg.